appengine/standard/endpoints/backend/echo-v1_swagger.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
It is possible, I think. This swagger is autogenerated from the endpoints service by a new tool. I don't think it has an option to generate yaml, but I can convert it if that's preferred.